lib/Access.php

```diff
@@ -1369,12 +1369,17 @@ private function prepareSearchTerm($term) {
 		$config = \OC::$server->getConfig();
 
 		$allowEnum = $config->getAppValue('core', 'shareapi_allow_share_dialog_user_enumeration', 'yes');
+		$allowMedialSearches = $config->getSystemValue('user_ldap.medial_search.enable', false);
```

Comment:
Do you think there will be other options related to medial search ? If not, just call it `user_ldap.enable_medial_search`.

Please also add it to config.sample.php and specify that it is experimental